feature-libs/product-configurator/rulebased/core/model/configurator.model.ts

```diff
@@ -105,6 +111,7 @@ export namespace Configurator {
     hideBasePriceAndSelectedOptions?: boolean;
     immediateConflictResolution?: boolean;
     newConfiguration?: boolean;
+    comments?: Comment[];
```

Comment:
as discussed we should store this outside the configuration, as it is not related to it.
for now we can just put it into a property of the view